### PR TITLE
Document the Export Set/Unset commands.

### LINKED ISSUE #6963: Document `Export Set/Unset`
https://github.com/coq/coq/pull/6923 needs documentation

### PR DESCRIPTION
**Pull request by ppedrot**
Fixes #6963.

### CODE REVIEW

File: doc/sphinx/proof-engine/vernacular-commands.rst [hunk 1]
```diff
@@ -128,6 +133,11 @@ This command switches :n:`@flag` off. The original
 state of :n:`@flag` is *not* restored at the end of the module. Additionally,
 if set in a file, :n:`@flag` is switched off when the file is `Require`-d.
 
+.. cmdv:: Export Set @flag.
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/vernacular-commands.rst]: Copy-paste typo: you meant `Unset`

File: doc/sphinx/proof-engine/vernacular-commands.rst [hunk 2]
```diff
@@ -180,6 +195,12 @@ value. The original state of :n:`@option` is *not* restored at the end of the
 module. Additionally, if unset in a file, :n:`@option` is reset to its
 default value when the file is `Require`-d.
 
+.. cmdv:: Export Unset @option.
+
+This command resets :n:`@option` to its default
+value. The original state of :n:`@option` is restored at the end of the
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/vernacular-commands.rst]: That's not correct: options are sometimes on by default, and sometimes off by default.

Comment (ppedrot) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: WDYM? The phrasing seems correct to me. It's an adaptation of what is written above already.

File: doc/sphinx/proof-engine/vernacular-commands.rst (hunk 1 above)
Comment (ppedrot) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: Will fix it.

File: doc/sphinx/proof-engine/vernacular-commands.rst (hunk 2 above)
Comment (Zimmi48) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: Well, you are right. The text above is incorrect as well.
`Unset` does not reset to the default value. It unsets the option. Sometimes, options are set by default and must be unset to see a different result.

Comment (Zimmi48) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: That being said, your PR is not making the reference manual worse, so this remark shouldn't be blocking it.

Comment (maximedenes) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: Why not fix it while we are at it?

Comment (maximedenes) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: Btw this whole flag/option distinction seems very strange to me. We use the terminology option for flags in the rest of the manual.

Comment (maximedenes) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: @ppedrot Could you fix the documentation ASAP please?

Comment (ppedrot) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: @maximedenes I don't know what to write. Solely writing *unsets the option* sounds completely non-informative to me.

Comment (maximedenes) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: `turns off`?

Comment (maximedenes) [reply doc/sphinx/proof-engine/vernacular-commands.rst]: It will be better that the current documentation, which is wrong, don't you think?

**Comment by ppedrot** [comment]
OK, I rephrased using *turns off*.

**Comment by maximedenes** [comment]
Sorry I just read again. If "options" are non-boolean options ("flags" being boolean ones), then I can't reproduce what the doc describes. It seems to me that `Unset foo` fails on such an option. @ppedrot did you actually test it?

**Comment by ppedrot** [comment]
This discussion is slightly getting off-topic. The original PR was just expanding on the wrong documentation, if we discuss about the proper fix it might take too long...

**Comment by maximedenes** [comment]
Sorry, just noticed: can you put the description of each of these options in the `cmdv` environment? That requires indenting the text.

**Comment by ppedrot** [comment]
@maximedenes None of the uses of `cmdv` in that file are indented. Am I missing something? I don't know the syntax so I cannot really help.

**Comment by ppedrot** [comment]
Should be OK now, I guess?

**Review by maximedenes — APPROVED** [review]
Thanks!

**Pull request merged**